### PR TITLE
fix(ui5-toolbar): remove unnecessary styles

### PR DESCRIPTION
**Pull request by dobrinyonkov**
This change removes unnecessary styles of the Toolbar.css file, related to the background of the Toolbar.
As we decided not to implement `desing` and `styling` property, some styles were not removed with the initial commit of the toolbar component.

Additionally, a risky override is removed of the color of the toolbar items.

### CODE REVIEW

File: packages/main/src/themes/Toolbar.css [hunk 1]
```diff
@@ -39,10 +26,6 @@
     width: 100%;
 }
 
-.ui5-tb-item[design="Transparent"] {
```
Comment (nnaydenow) [inline packages/main/src/themes/Toolbar.css:42]: I think that this is related to the slotted items. Are we sure that we want to remove it?

File: packages/main/src/themes/Toolbar.css [hunk 2]
```diff
@@ -6,23 +6,10 @@
 	justify-content: flex-end;
 	padding: 0 var(--_ui5-toolbar-padding-left) 0 var(--_ui5-toolbar-padding-right);
 	box-sizing: border-box;
-	background: var(--ui5-toolbar-Background);
 	color: var(--sapButton_TextColor);
```
Comment (nnaydenow) [inline packages/main/src/themes/Toolbar.css:9]: Also do we need this color here?

**Review by nnaydenow — COMMENTED** [review]
At all if I remove these two styles the component looks like:

image

**Comment by dobrinyonkov** [comment]
> At all if I remove these two styles the component looks like:
> 
> image

This seems to look different for me, overall those color overrides were never part of the fiori VD spec. They were part of the POC that we forgot to remove. 

image

File: packages/main/src/themes/Toolbar.css (hunk 2 above)
Comment (dobrinyonkov) [reply packages/main/src/themes/Toolbar.css:9]: Correct, the color: `var(--sapButton_TextColor);` is also an extra.

**Review by PetyaMarkovaBogdanova — APPROVED** [review]

**Pull request merged**